atlp-rwanda/vikings-ec-bn
# #184347023 implement 2 factor authentication

### What does this PR do?

Implement 2 Factor Authentication for sellers

### Description of Task to be completed?

Users of type seller should verify their account with a one-time password sent to their email every time they try to log in.

### How should this be manually tested?

1. Checkout to this branch: `feature/2fa-184347023`
2. Run the command `npm install`
3. Run `npm run seeding`
4. Open the browser and visit http://localhost:5000/api-docs/#/
5. Register a user with a seller role and verify them
6. Try to log in with the credentials of that user
7. Test functionalities of users/login/auth

#### What are the relevant pivotal tracker stories?

[#184347023](https://www.pivotaltracker.com/story/show/184347023)

#### Screenshots
![image](https://user-images.githubusercontent.com/73304870/219005536-e2733436-d0c4-4b02-ba5b-71e510f68709.png)
